homeassistant/components/airly/sensor.py

```diff
@@ -68,7 +68,7 @@ class AirlySensorEntityDescription(SensorEntityDescription):
     AirlySensorEntityDescription(
         key=ATTR_API_CAQI,
         icon="mdi:air-filter",
-        name=ATTR_API_CAQI,
+        name="Common Air Quality Index",
```

Comment:
Our naming standards use sentence style casing.

```suggestion
        name="Common air quality index",
```

Reply:
Due to the length, I reverted the old name `CAQI`.